tests/tasks/folio/test_build.py

```diff
@@ -189,7 +189,7 @@ def test_instance_type_id(mock_requests_okapi):  # noqa: F811
         okapi_uri, "sul", "test_user", "asdfdsfa"
     )
     instance_type_id = _instance_type_id(
-        values=[["text"]], folio_client=folio_client_mock
+        values=[["Text"]], folio_client=folio_client_mock
```

Comment:
or put another way, this seems like it was changed to exercise the code change that was made, but i can't figure out where in the test the comparison value is set.